data-plane/python-bindings/srpc/src/srpc/server.py

```diff
@@ -0,0 +1,138 @@
+# Copyright AGNTCY Contributors (https://github.com/agntcy)
+# SPDX-License-Identifier: Apache-2.0
+
+import asyncio
+import datetime
+
+
+import click
+
+import slim_bindings
+
+from .rpc import Rpc
```

Comment:
its best practise to always us absolute imports even within the same package like:

```
from srpc.rpc import Rpc
```